### PR TITLE
index: fix scripts not loading on HTTPS

### PR DESCRIPTION
**Pull request by cyphar**
We must load scripts over HTTPS, otherwise browsers block us from
loading the scripts we ned to run if the site is loaded over HTTPS.

Fixes: #203
Closes: #202
Signed-off-by: Aleksa Sarai <cyphar@cyphar.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
 	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
 	<meta name="keywords" content="cv, resume, hResume, github, community, free, fun, nice, useful" />
 	<meta name="description" content="GitHub resumes, generated by the community, for the community." />
-	<link rel="stylesheet" type="text/css" href="http://yui.yahooapis.com/2.7.0/build/reset-fonts-grids/reset-fonts-grids.css" media="all" /> 
+	<link rel="stylesheet" type="text/css" href="https://yui.yahooapis.com/2.7.0/build/reset-fonts-grids/reset-fonts-grids.css" media="all" />
```
Comment (Walkman100) [inline index.html]: image

File: index.html [hunk 2]
```diff
@@ -5,10 +5,10 @@
...
 	<link rel="stylesheet" type="text/css" href="css/resume.css" media="all" />
 	<link rel="stylesheet" type="text/css" href="css/print.css" media="print" />
-	<script type="text/javascript" src="http://code.jquery.com/jquery-1.8.1.min.js"></script>
+	<script type="text/javascript" src="https://code.jquery.com/jquery-1.8.1.min.js"></script>
```
Comment (Walkman100) [inline index.html]: This has a valid https certificate, should at least be merged in without further ado.

File: index.html (hunk 1 above)
Comment (auscompgeek) [inline index.html]: Looks like Yahoo's CDN uses `yui-s.yahooapis.com` for HTTPS connections. Weird.

**Comment by cyphar** [comment]
@Walkman100 fix'd.

**Comment by phanimahesh** [comment]
:+1:  Was just about to submit a PR for this when I found this.

**Comment by jaredallard** [comment]
:+1: to this, since I use HTTPs (literally) Everywhere!

**Comment by nootanghimire** [comment]
Just submitted a PR and found this. :+1:  to the PR. but why isn't no one merging it?

**Comment by gdude2002** [comment]
Sure, I'll give you guys my :+1:, surprised this isn't already merged

**Comment by cyphar** [comment]
@gdude2002 

> Actually, it would be better to use `//` instead of `https://` here, imo.

Yahoo doesn't support `http://` and `https://` for their CDN with the same url.

**Comment by gdude2002** [comment]
@cyphar Not *all* cases, but most, yeah. It's always the more secure option but https isn't always available unfortunately..

Also, that's a pretty weird thing for a CDN to do, lol. But if we're stuck with it, I guess..

**Comment by kurtisnelson** [comment]
:+1: HTTPS everywhere currently breaks the site

**Comment by Blaisorblade** [comment]
:+1:, same problem here.

> > EDIT: Actually, it would be better to use // instead of https:// here, imo.

> Yahoo doesn't support http:// and https:// for their CDN with the same url. Also, HTTPS is better in all cases.

OK, but `//` would work for JQuery (confirmed here: https://jquery.com/download/).

**Comment by cyberhck** [comment]
Why is this not merged? Also I've sent two fixes in a PR nobody is responding to that either.

**Comment by Blaisorblade** [comment]
> Why is this not merged? Also I've sent two fixes in a PR nobody is responding to that either.

@davidcoallier, given you have not pushed commits here for the last two years: do you plan to resume maintaining this project? Otherwise, do you plan to pass on maintainership (if somebody steps in), or to allow forking by adding an open source _license_ (see http://choosealicense.com/)?

Currently, we're technically allowed to fork this repo (by GitHub ToS, as explained [here](http://choosealicense.com/no-license/)), but we aren't allowed to modify it.

**Comment by u2** [comment]
Duplicate #190 .Discuss at #182 . There are two many same issues and PR, it make me confused.

**Comment by cyphar** [comment]
@u2 There, I've rebased.

**Pull request merged**